pt-BR/mail/chrome/messenger/messenger.properties

```diff
@@ -485,426 +485,426 @@ copyToFolderAgain=Copiar de novo para “%1$S”
-confirmSaveMessageEdits=Are you sure you want to replace the original message with these changes?
-skipFutureConfirmSaveMessageEditsPrompts=Do not show me this dialog box again.
+confirmNavigateAwayFromMessageEditingTitle=Descartar a edição atual?
+confirmNavigateAwayFromMessageEditing=Tem certeza de que deseja sair desta mensagem?\n\nVocê possui alterações não salvas nesta mensagem.\n\nPressione OK para continuar ou Cancelar para ficar na mensagem atual.
+alertEditOneMessage=Você só pode editar uma mensagem de cada vez.
+confirmSaveMessageEdits=Tem certeza de que deseja alterar a mensagem original com essas mudanças?
+skipFutureConfirmSaveMessageEditsPrompts=Não exibir esta janela novamente.
 
 # pbQuickReply
-confirmNavigateAwayFromQuickReplyTitle=Discard the current reply?
-confirmNavigateAwayFromQuickReply=The content within your current reply will be lost.
+confirmNavigateAwayFromQuickReplyTitle=Descartar a resposta atual?
+confirmNavigateAwayFromQuickReply=O conteúdo dentro da resposta atual será perdido.
 
 # dataPane.js
-dataPaneShowMoreLink=more...
+dataPaneShowMoreLink=mais...
 
 #pbDeliciousService.js
 delicious.loginDialog.title=del.icio.us
-delicious.loginDialog.check.text=Use Password Manager to remember this password.
-delicious.loginDialog.text=Enter your username and password for del.icio.us.
-delicious.loginDialog.nick.label=Username
-delicious.loginDialog.password.label=Password
-delicious.loginDialog.link.label=Create an account
-delicious.throttled.text=You have been throttled for making too many connections to del.icio.us. Please try again later.
-delicious.add_post.title=Bookmarking on del.icio.us:
+delicious.loginDialog.check.text=Use o Gerenciador de Senhas para memorizar esta senha.
+delicious.loginDialog.text=Digite o nome de usuário e a senha de del.icio.us.
+delicious.loginDialog.nick.label=Nome de usuário
+delicious.loginDialog.password.label=Senha
+delicious.loginDialog.link.label=Criar uma conta
+delicious.throttled.text=Você foi limitado por realizar muitas conexões para o delicio.us. Por favor, tente novamente mais tarde.
+delicious.add_post.title=Tonar del.icio.us favorito:
 
 #pbFriendFeedService.js
 friendfeed.service.name=FriendFeed
-friendfeed.service.description=The FriendFeed service will allow you to post to your FriendFeed account.
-friendfeed.loginDialog.title=FriendFeed Remote Login
-friendfeed.loginDialog.check.text=Remember Me
-friendfeed.loginDialog.text=Enter your nickname and remote key for FriendFeed.
-friendfeed.loginDialog.nick.label=Friendfeed nickname or email
-friendfeed.loginDialog.password.label=Remote key
-friendfeed.loginDialog.link.label=Find your Remote Key
-friendfeed.throttled.text=You have been throttled for making too many connections to FriendFeed. Please try again later.
-friendfeed.login_failed.text=Failed to log in, please try again.
+friendfeed.service.description=O serviço FriendFeed permitirá que você publique na sua conta doFriendFeed.
```

Comment:
CORRECTIONS NEEDED it should be friendfeed.service.description=O serviço FriendFeed permitirá que você publique na sua conta do FriendFeed.